File: 3- operacoes aritmeticas/operacoes.go
package main

import "fmt"

func main() {
	var a, b float64
	fmt.Println("Digite dois números: ")
	fmt.Scan(&a, &b)
	fmt.Printf("Soma: %.2f\n", a+b)
	fmt.Printf("Subtração: %.2f\n", a-b)
	fmt.Printf("Multiplicação: %.2f\n", a*b)
	fmt.Printf("Divisão: %.2f\n", a/b)
}

File: 3- operacoes aritmeticas/operacoes.c
#include <stdio.h>

int main() {
    float a, b;
    printf("Digite dois números: ");
    scanf("%f %f", &a, &b);
    printf("Soma: %.2f\n", a + b);
    printf("Subtração: %.2f\n", a - b);
    printf("Multiplicação: %.2f\n", a * b);
    printf("Divisão: %.2f\n", a / b);
    return 0;
}
